assets/device-all/templatecnn.h

```diff
@@ -29,8 +29,10 @@ typedef int16_t q15_t;
 #define SYS_START LED_On(0)
 #define SYS_COMPLETE LED_Off(0)
 
-/* Unload data from accelerator and run software SoftMax */
+/* Run software SoftMax on unloaded data */
 void softmax_q17p14_q15(const q31_t * vec_in, const uint16_t dim_vec, q15_t * p_out);
```

Comment:
Why is **vec_in** declared as a const?

Reply:
Because the non-shift version does not modify the input data.